mlflow/pyfunc/scoring_server/__init__.py

```diff
@@ -431,6 +450,8 @@ def init(model: PyFuncModel):
 
     app = flask.Flask(__name__)
     input_schema = model.metadata.get_input_schema()
+    # set the environment variable to indicate that we are in a serving environment
+    os.environ["_MLFLOW_IS_SERVING_ENVIRONMENT"] = "true"
```

Comment:
nit: can't we use _MLFLOW_IS_SERVING_ENVIRONMENT instead of hardcoding env var name?